spec/features/users/sign_up_spec.rb

```diff
@@ -148,4 +148,26 @@
   it_behaves_like 'csrf error when acknowledging personal key', :oidc
   it_behaves_like 'creating an account with the site in Spanish', :saml
   it_behaves_like 'creating an account with the site in Spanish', :oidc
+
+  it 'allows a user to choose TOTP as 2FA method during sign up' do
```

Comment:
This looks good, but I wonder if we should include the process for setting up the auth app?

Reply:
We absolutely should. This is not working properly.